app/src/main/java/org/mifos/selfserviceapp/ui/fragments/SavingAccountsTransactionFragment.java

```diff
@@ -144,8 +145,7 @@ public void showErrorFetchingSavingAccountsDetail(String message) {
 
     @Override
     public void showFilteredList(List<Transactions> list) {
-        Toast.makeText(getContext(), getContext().getResources().getText(R.string.filterd)
-                , Toast.LENGTH_SHORT).show();
+        Toaster.show(rootView, "" + getContext().getResources().getText(R.string.filterd));
```

Comment:
I think this ` Toaster.show(getActivity(), getString(R.string.filterd)
                , Toast.LENGTH_SHORT).show();` would work too. 
Make the change accordingly.